addons/folo/jaxrs/src/main/java/org/commonjava/indy/folo/bind/jaxrs/FoloAdminResource.java

```diff
@@ -345,4 +353,58 @@ public Response importReport( final @Context UriInfo uriInfo, final @Context Htt
         return types;
     }
 
+    @ApiOperation( "Batch delete files uploaded through FOLO trackingID under the given storeKey." )
+    @ApiResponse( code=200, message = "Batch delete operation finished." )
+    @ApiImplicitParam( name = "body", paramType = "body",
+                    value = "JSON object, specifying trackingID and storeKey, with other configuration options",
+                    required = true, dataType = "org.commonjava.indy.model.core.DemoteRequest" )
+    @Path( "/batch/demote" )
+    @POST
+    public Response doDelete( @Context final UriInfo uriInfo, final DemoteRequest request )
```

Comment:
I haven't found a matching method in the java client. I suppose it was forgotten.

Reply:
There is one in foloAdminClientModule, https://github.com/Commonjava/indy/pull/1557/files#diff-b35f8f00876e08a42e9c82c32e08824c

So you don't use the foloAdminClientModule, do you ? 

I will try to keep it simple in client module as well, thanks.

Reply:
Thanks for pointing me at it, I had to miss it because it is the first change :-) Sorry.

Reply:
Using the foloAdminClientModule would be preferred for the batch delete, if we can rely on the tracking reports' contents. That would allow us that PNC Cleaner doesn't have to read the list of paths from DB but would pass just the tracking ID instead.

Reply:
Yeah, glad to see this can easy your work. :)